src/main/java/com/puppycrawl/tools/checkstyle/Checker.java

```diff
@@ -426,36 +386,40 @@ public void fireErrors(String fileName, SortedSet<Violation> errors) {
      */
     @Override
     public void fireFileFinished(String fileName) {
-        final String stripped = CommonUtil.relativizePath(basedir, fileName);
-        final AuditEvent event = new AuditEvent(this, stripped);
         for (final AuditListener listener : listeners) {
-            listener.fileFinished(event);
+            listener.fileFinished(
+                new AuditEvent(this, CommonUtil.relativizePath(basedir, fileName)));
         }
     }
 
     @Override
     protected void finishLocalSetup() throws CheckstyleException {
-        final Locale locale = new Locale(localeLanguage, localeCountry);
-        LocalizedMessage.setLocale(locale);
+        LocalizedMessage.setLocale(new Locale(localeLanguage, localeCountry));
 
         if (moduleFactory == null) {
             if (moduleClassLoader == null) {
                 throw new CheckstyleException(getLocalizedMessage("Checker.finishLocalSetup"));
             }
-
-            final Set<String> packageNames = PackageNamesLoader
-                    .getPackageNames(moduleClassLoader);
-            moduleFactory = new PackageObjectFactory(packageNames,
-                    moduleClassLoader);
+            moduleFactory = new PackageObjectFactory(
+                PackageNamesLoader.getPackageNames(moduleClassLoader),
+                moduleClassLoader);
```

Comment:
higher `cohesion` as its tighter together, building the block pattern, packing up.